### DISCUSSION TITLE
css not working

### DISCUSSION THREAD

**Discussion by Tybeos** [thuliteio/doks#963] [Q&A] [answered]
Hello,

When I run "npm run build" and after that pull all files from public directory to FTP server css doesn't render, only raw HTML website.
Other Go Hugo website works on that FTP server fine. 

"npm test" it has no error

I set baseURL to propper adress.
also set canonifyURLs = true

I have read the entire documentation and I have no idea what I'm doing wrong.
I saw that some people here in the discussion also had a same problem, but none of the solutions worked for me.
Can you help me please?

**Comment by h-enk** [comment]
Please share your repo + live site url

**Comment by Tybeos** [comment]
live site url [bridgeofukraine](http://bridgeofukraine.org/en/)

repo: https://github.com/Tybeos/doks

Image how should website looks
https://github.com/Tybeos/doks/blob/master/images/screen.png?raw=true

I did some testing, I think it's related to using svg images in \layouts\index.html with custom class add in \assets\scss\components\ _images.scss

**Comment by h-enk** [comment, marked as answer]
Looks like an HTTPS issue on your server. When I visit https://bridgeofukraine.org, I get "This site can’t be reached". When I visit http://bridgeofukraine.org, open DevTools and go to the Console tab, I get the following error messages:

![Snag_5b43105](https://user-images.githubusercontent.com/3902872/209308163-42937f40-f879-40bd-8ce1-6ef5e70f0080.png)

👉 Make sure your site can only be accessed through HTTPS, and force those connections.

What I did to test/verify your issue (could not reproduce it):
- I've cloned your repo, 
- updated `baseurl = "https://bridgeofukraine.org/"` to `baseurl = "https://fascinating-crumble-c9c280.netlify.app/"`,
- ran `npm run build`, and 
- uploaded the `public` folder to [Netlify Drop](https://app.netlify.com/drop)

Result: https://fascinating-crumble-c9c280.netlify.app — works like a charm

Note, I will leave this site live for you to see and will remove it when your site is live succesfully

**Comment by Tybeos** [comment]
Thank you for the reply, help and your time. It's already working! I think the problem was that I didn't have an activated ssl certificate.
https://bridgeofukraine.org/
